src/components/KExternalLink/KExternalLink.vue

```diff
@@ -61,17 +47,18 @@ const iconColor = computed((): string => isMouseOver.value ? 'var(--KExternalLin
 
 .k-external-link {
   align-items: center;
-  color: var(--KExternalLinkColor, color(blue-500));
+  color: var(--KExternalLinkColor, var(--kui-color-text-primary, $kui-color-text-primary));
   display: inline-flex;
-  font-weight: 400;
+  font-weight: var(--kui-font-weight-regular, $kui-font-weight-regular);
   text-decoration: none;
 
   &:hover {
-    color: var(--KExternalLinkColorHover, color(blue-600));
+    color: var(--KExternalLinkColorHover, var(--kui-color-text-primary-strong, $kui-color-text-primary-strong));
   }
 
   .kong-icon {
-    margin-left: var(--spacing-xs);
+    margin-left: var(--spacing-xs, var(--kui-space-40, $kui-space-40));
   }
 }
 </style>
+@/utilities/urls
```

Comment:
remove this?

Reply:
Stupid VS Code import autoupdate, removed